src/connection/index.ts

```diff
@@ -20,6 +20,18 @@ export enum ConnectionType {
   NETWORK = 'NETWORK',
   GNOSIS_SAFE = 'GNOSIS_SAFE',
 }
+interface WalletInfo {
```

Comment:
This isn't used, and it isn't exported. Can this be removed, or should it be exported?

Reply:
It'll be used when we import the new wallet component. I'll remove for now and it'll get added as part of the PR for the new wallet